gbpservice/nfp/configurator/agents/generic_config.py

```diff
@@ -6,9 +6,15 @@
 from gbpservice.nfp.core import main
 import os
 from gbpservice.nfp.configurator.lib import utils
-
+from gbpservice.nfp.core import poll as nfp_poll
 LOG = logging.getLogger(__name__)
 
+SUCCESS = 'SUCCESS'
```

Comment:
Some of these constants are common to base driver and generic config. Why not add these to constants.py file in lib directory.